### ISSUE TITLE
[BUG] Modifications to custom configuration files cannot take effect immediately

### ISSUE THREAD

**Issue by weizhuangzhi** [bunkerity/bunkerweb#712]
### What happened?

Modifying the custom configuration file does not take effect immediately. The custom configuration will not take effect until the `systemctl restart bunkerweb `command is executed.

### How to reproduce?

On the WebUI configuration page, accessing the website configuration after adding, modifying, or deleting custom configurations to configs/modsec-crs/mysite.net/ does not work. For example, if I add an exclusion rule: SecRuleRemoveById 932115, the expected effect can only be achieved after restarting the bunkerweb service. I am not sure whether this problem exists in the release version. My impression seems to be that it takes effect immediately. I am currently using the dev version and have this problem.

### Configuration file(s) (yaml or .env)

```YAML
SecRuleRemove.conf
......
SecRuleRemoveById 932115
......
```

### Relevant log output

```shell
nothing
```

### BunkerWeb version

1.5.2

### What integration are you using?

Linux

### Linux distribution (if applicable)

fedora38

### Removed private data

- [X] I have removed all private data from the configuration file and the logs

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by TheophileDiot** [comment]
Hi @weizhuangzhi, thank you for opening this issue. Are you talking about modifying the file manually in the file manager ?

**Comment by TheophileDiot** [comment]
I'll try this out and let you know, thank you again.

**Comment by TheophileDiot** [comment]
@weizhuangzhi the file has to have at least 5 characters, is this the issue why the file isn't being created ?

**Comment by weizhuangzhi** [comment]
![image](https://github.com/bunkerity/bunkerweb/assets/4264162/61838ae6-f2bf-47ec-98c6-e17f1acce2ca)
Is this the file you're talking about? The file is created correctly, but it feels like nginx is not reloaded, not sure.

**Comment by TheophileDiot** [comment]
@weizhuangzhi, I tried it out on the `dev` branch, and it's working fine. I'll try on the `1.5.2` one and let you know.

**Comment by TheophileDiot** [comment]
Mine is created successfully 🤔
```
35e92b90a0e0:/data/configs# tree -L 3
.
├── default-server-http
├── default-server-stream
├── http
├── modsec
├── modsec-crs
│   └── www.example.com
│       └── rule.conf
├── server-http
├── server-stream
└── stream

9 directories, 1 file
```

**Comment by weizhuangzhi** [comment]
The file can be created successfully, but the configuration items in the file do not take effect.

The expected effect is that the website no longer checks the problem of 932115 (but in fact it is still checking)
SecRuleRemoveById 932115

**Comment by TheophileDiot** [comment]
Oh, I see, Indeed. I'll check what's the issue and let you know.

**Comment by TheophileDiot** [comment]
The problem was indeed due to a problem with the chair-keyboard interface. The next commit will fix this.

**Comment by weizhuangzhi** [comment]
Thank you for your reply.
I use Google Translate to translate the problem description. Sometimes I can't express the problem accurately, which causes trouble for you. I'm sorry.

**Comment by TheophileDiot** [comment]
No problem, thank you for your huge contributions by the way !

**Comment by weizhuangzhi** [comment]
[#712] Fix custom configuration changes not taking effect immediately

After testing, the configuration changes will take effect immediately, which is great.

**Comment by fl0ppy-d1sk** [comment]
Hello @weizhuangzhi,

It should be fixed in the last 1.5.3 release. Don't hesitate to open a new issue if needed.
